cmd/ocitools/generate.go

```diff
@@ -382,3 +406,15 @@ func setupLinuxNamespaces(g generate.Generator, needsNewUser bool, nsMaps map[st
 		g.AddOrReplaceLinuxNamespace(nsName, nsPath)
 	}
 }
+
+func onlyExportFlagSpecified(context *cli.Context) bool {
+	onlyExportFlags := []bool{
+		context.Bool("seccomp-only"),
+	}
+	for _, flag := range onlyExportFlags {
+		if flag {
+			return true
```

Comment:
On Thu, Jul 21, 2016 at 03:13:27PM -0700, Antonio Murdaca wrote:
> You can have a struct to configure optional arguments to this Save
> function (or SaveToFile). I'm assuming those functions are from the
> libraries and in this case that context (coming from a cli package)
> should not clutter the library.

Who cares where the Context comes from?  You'd just want to document
(on Save) the properties it looked at and (on SaveToFile) the fact
that it was a thin wrapper around Save.  A config struct accomplishes
the same result, but it feels kludgier to me.  But if a config struct
is more idiomatic Go than a Context argument, that's more important
than how I feel ;).